server/server_test.go

```diff
@@ -57,6 +57,11 @@ func TestT(t *testing.T) {
 	if !reflect.DeepEqual(defaultConfig, globalConfig) {
 		t.Fatalf("%#v != %#v\n", defaultConfig, globalConfig)
 	}
+
+	config.UpdateGlobal(func(conf *config.Config) {
+		conf.TiKVClient.AsyncCommit.SafeWindow = 0
+		conf.TiKVClient.AsyncCommit.AllowedClockDrift = 0
```

Comment:
should we set the `tidb_enable_async_commit=0 and tidb_enable_async_commit=0` ?

Reply:
This change is enough.
In a testing environment (and not intend to testing for that specific feature), it's safe to set this config.
Or otherwise we can set the lease to 0, you can see here

https://github.com/pingcap/tidb/blob/e7d73718aaadecc7925ff950699a20ea1d33c8ac/ddl/reorg.go#L204-L209

Reply:
I actually prefer to set lease to 0, which has a smaller impact. These two config variables will affect client-go too. Will that make tests unstable? I don't know...

Reply:
both fine to me

Reply:
Change to set lease @xhebox